src/types/json.h

```diff
@@ -173,6 +173,22 @@ struct JsonValue {
     return types;
   }
 
+  Status Toggle(std::string_view path) {
+    try {
+      jsoncons::jsonpath::json_replace(value, path, [&](const std::string & /*path*/, jsoncons::json &val) {
+        bool is_boolean = val.is_bool() && !val.empty();
```

Comment:
May I ask why we check `empty()` here? Seems `is_bool()` is enough to me.

Reply:
Thanks for pointing out. will fix